helpick-service/src/main/java/com/infoshare/service/VolunteerService.java

```diff
@@ -58,4 +59,25 @@ public boolean registerNewVolunteer(String name, String location, String email,
         }
     }
 
+    public boolean editVolunteerData(String name, String location, String email, String phone, TypeOfHelp typeOfHelp,
+                                     boolean availability, UUID uuid) {
+        Optional<Volunteer> volunteer = db.getVolunteer(uuid);
+        if (volunteer.isPresent()) {
+            volunteer.get().setName(name);
+            volunteer.get().setLocation(location);
+            volunteer.get().setPhone(phone);
+            volunteer.get().setTypeOfHelp(typeOfHelp);
+            volunteer.get().setAvailable(availability);
+            volunteer.get().setEmail(email);
+            db.saveVolunteerWithUuid(volunteer.get());
```

Comment:
Tutuaj tak jak wspominalam wczesniej, mozna to zamienic na: 
```suggestion
           db.saveVolunteer(volunteer.get());
```

Reply:
jak wyżej :)